modules/nodes-pods-autoscaling-about.adoc

```diff
@@ -0,0 +1,144 @@
+|===
+
+|Metric |Description |API version
+
+|CPU utilization
+|Percentage of the requested CPU
+|`autoscaling/v1`, `autoscaling/v2beta1`
+
+|Memory utilization
+|Percentage of the requested memory.
+|`autoscaling/v2beta1`
+|===
+
+[IMPORTANT]
+====
+For memory-based autoscaling to work, memory usage must increase and decrease
+proportionally to the replica count. On average:
+
+* An increase in replica count must lead to an overall decrease in memory
+(working set) usage per-pod.
+* A decrease in replica count must lead to an overall increase in per-pod memory
+usage.
+
+Use the OpenShift web console to check the memory behavior of your application
+and ensure that your application meets these requirements before using
+memory-based autoscaling.
+====
+
+
+== Understanding how to create a horizontal pod autoscaler
+
+How you create the autoscaler depends on whether you want to scale for CPU or memory utilization.
+
+* For CPU utilization You create the horizontal pod autoscaler object for CPU utilization. 
+The resulting horizontal pod autoscaler object is defined differently 
+when using the `autoscaling/v1` version or the the `autoscaling/v2beta1` version of the horizontal pod autoscaler:
++
+.Horizontal Pod Autoscaler Object Definition with the `autoscaling/v1`
+[source,yaml,options="nowrap"]
+----
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: frontend <1>
+spec:
+  scaleTargetRef:
+    kind: DeploymentConfig <2>
+    name: frontend <3>
+    apiVersion: apps/v1 <4>
+    subresource: scale
+  minReplicas: 1 <5>
+  maxReplicas: 10 <6>
+  cpuUtilization:
+    targetCPUUtilizationPercentage: 80 <7>
+----
+<1> The name of this horizontal pod autoscaler object
+<2> The kind of object to scale
+<3> The name of the object to scale
+<4> The API version of the object to scale
+<5> The minimum number of replicas to which to scale down
+<6> The maximum number of replicas to which to scale up
+<7> The percentage of the requested CPU that each pod should ideally be using
++
+.Horizontal Pod Autoscaler Object Definition with the `autoscaling/v2beta1`
+[source,yaml,options="nowrap"]
+----
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: hpa-resource-metrics-cpu <1>
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1 <2>
+    kind: ReplicationController <3>
+    name: hello-hpa-cpu <4>
+  minReplicas: 1 <5>
+  maxReplicas: 10 <6>
+  metrics:
+  - type: Resource
+    resource:
+      name: cpu
+      targetAverageUtilization: 50 <7>
+----
+<1> The name of this horizontal pod autoscaler object
+<2> The API version of the object to scale
+<3> The kind of object to scale
+<4> The name of the object to scale
+<5> The minimum number of replicas to which to scale down
+<6> The maximum number of replicas to which to scale up
+<7> The average percentage of the requested CPU that each pod should be using
+
+* For memory utilization, you create the autoscaler using a YAML file. Optionally,
```

Comment:
if you want to keep the "you", which i don't prefer, just make sure you keep it in sync with "For CPU utilization" above.